### ISSUE TITLE
Issue creating DAL2 MVC Module

### ISSUE THREAD

**Issue by jayslife** [ChrisHammond/DNNTemplates#68]
Using Visual Studios 2019 Professional Version 16.7.2
Running in Admin mode
My Website was orginally setup using nvQuickSite, but then moved to C:\websites\dnndev.me
IIS site was renamed to dnndev.me and resolves via http://dnndev.me

I have tried to follow all the videos published and still keep coming across this issue creating a new module.

![image](https://user-images.githubusercontent.com/9482630/135761421-a2a90fd5-132f-4433-9f3e-93b6066263e3.png)

Please advise what the issue could be.

**Comment by ChrisHammond** [comment]
Sent this in an email as well:

Typically when people have problems with the templates it is one of two causes.

Either Visual Studio has created virtual directories that need removed, or you have something off with the Path. It looks like you've got the path squared away based on the screenshot in your issue on Github, but it might be worth double checking things. You also might check if you can create any other templates, or if it is just the MVC template.

One thing you might try is removing the Period from the project/solution name, I don't know if that will cause an issue, but worth a shot. I am currently doing some system maintenance and won't have a VM squared away for a few days to do any testing on my side.

**Comment by jayslife** [comment]
I tried all the above suggestions and was unable to get v10.0.2 of the templates to work. I uninstalled v10.0.2 from my visual studio 2019 and manually downloaded v10.0.0 vsix from git, after installing v10.0.0 I am now able to create templates. My assumption is that the SetupWizard.dll included in v10.0.2 is corrupt and fails on fresh installs. @ChrisHammond  thanks again for your support and replay to my issue.  

Note: **Automatically Update this extension** get checked by default, after installing v10.0.0, upon first launch you will need to go to installed extensions area in Visual Studio and uncheck this box to keep v10.0.0 installed.
